.gitignore

```diff
@@ -4,3 +4,4 @@
 /settings.json.code-workspace
 **swp
 /Cargo.lock
+book/html
```

Comment:
this will need to be updated to reflect the proper book location (`src/docs/goose-book/book`)

Reply:
Resolving as can't fix in 3-6 months.